fs/dafsserver/dafsserver.cpp

```diff
@@ -4396,18 +4401,20 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                     catch (IJSOCK_Exception *e)
```

Comment:
minor: This specialised catch could now be deleted.

Reply:
agree, will remove.